contracts/import.sol

```diff
@@ -8,4 +8,7 @@ import '@rsksmart/rif-relay-contracts/contracts/factory/CustomSmartWalletFactory
 import '@rsksmart/rif-relay-contracts/contracts/verifier/DeployVerifier.sol';
 import '@rsksmart/rif-relay-contracts/contracts/verifier/RelayVerifier.sol';
 import '@rsksmart/rif-relay-contracts/contracts/RelayHub.sol';
+import '@rsksmart/rif-relay-contracts/contracts/utils/UtilToken.sol';
+
+import '@rsksmart/rif-relay-contracts/contracts/interfaces/IForwarder.sol';
```

Comment:
What is the purpose of this file? It appears to me that it just bloats the tests with imports that individual test cases might not need?

Reply:
The purpose of this file is to include all the contracts we want to be deployed everytime we run the tests.
> imports that individual test cases might not need
Yep, it's true, but we're talking about integration/e2e tests. Those contracts will be deployed once for all the tests. Which approach would you suggest? Should we manually deploy the contracts in each file?

Reply:
nothing to do here, it's as @antomor says